src/Components/Feelings/Feelings.js

```diff
@@ -11,29 +12,29 @@ const Feelings = ({ setFeeling }) => {
-        <p className='feeling sad' onClick={selectFeeling}>Depressed</p>
-        <p className='feeling surprised' onClick={selectFeeling}>Surpised</p>
-        <p className='feeling surprised' onClick={selectFeeling}>Confused</p>
-        <p className='feeling surprised' onClick={selectFeeling}>Excited</p>
-        <p className='feeling bad' onClick={selectFeeling}>Bad</p>
-        <p className='feeling bad' onClick={selectFeeling}>Stressed</p>
-        <p className='feeling bad' onClick={selectFeeling}>Tired</p>
-        <p className='feeling fearful' onClick={selectFeeling}>Fearful</p>
-        <p className='feeling fearful' onClick={selectFeeling}>Anxious</p>
-        <p className='feeling fearful' onClick={selectFeeling}>Insecure</p>
-        <p className='feeling angry' onClick={selectFeeling}>Angry</p>
-        <p className='feeling angry' onClick={selectFeeling}>Distant</p>
-        <p className='feeling angry' onClick={selectFeeling}>Frustrated</p>
-        <p className='feeling disguisted' onClick={selectFeeling}>Disguisted</p>
-        <p className='feeling disguisted' onClick={selectFeeling}>Disappointed</p>
-        <p className='feeling disguisted' onClick={selectFeeling}>Awful</p>
-      </section>
+        <Link className='feelings-container' to='/what-should-you-do'>
+          <p className='feeling happy' onClick={selectFeeling}>Happy</p>
+          <p className='feeling happy' onClick={selectFeeling}>Content</p>
+          <p className='feeling happy' onClick={selectFeeling}>Optimistic</p>
+          <p className='feeling sad' onClick={selectFeeling}>Sad</p>
+          <p className='feeling sad' onClick={selectFeeling}>Lonely</p>
+          <p className='feeling sad' onClick={selectFeeling}>Depressed</p>
+          <p className='feeling surprised' onClick={selectFeeling}>Surpised</p>
+          <p className='feeling surprised' onClick={selectFeeling}>Confused</p>
+          <p className='feeling surprised' onClick={selectFeeling}>Excited</p>
+          <p className='feeling bad' onClick={selectFeeling}>Bad</p>
+          <p className='feeling bad' onClick={selectFeeling}>Stressed</p>
+          <p className='feeling bad' onClick={selectFeeling}>Tired</p>
+          <p className='feeling fearful' onClick={selectFeeling}>Fearful</p>
+          <p className='feeling fearful' onClick={selectFeeling}>Anxious</p>
+          <p className='feeling fearful' onClick={selectFeeling}>Insecure</p>
+          <p className='feeling angry' onClick={selectFeeling}>Angry</p>
+          <p className='feeling angry' onClick={selectFeeling}>Distant</p>
+          <p className='feeling angry' onClick={selectFeeling}>Frustrated</p>
+          <p className='feeling disguisted' onClick={selectFeeling}>Disguisted</p>
+          <p className='feeling disguisted' onClick={selectFeeling}>Disappointed</p>
+          <p className='feeling disguisted' onClick={selectFeeling}>Awful</p>
+        </Link>
```

Comment:
Like I mentioned earlier, I'd love to do a Feeling (singular) component and dynamically render each Feeling in the Feelings component. It would clean up this section a ton!